Update note on uniqueness of decoy integers (#259)
ietf-wg-spice/draft-ietf-spice-sd-cwt: draft-ietf-spice-sd-cwt.md

The integer could be used to look up the salt for the decoy deterministically, but does not impose any ordering.
When a decoy digest is requested in a map, the map *value* is always `null`.

{:aside}

> > Note: Requiring an integer that is unique per decoy within the entire CWT ensures that duplicate map keys are not needed (see {{Section 5.6 of CBOR}}).

Reviewer: Maybe?

```suggestion
> Note: Requiring an integer that is unique per decoy within the entire CWT ensures that there are no duplicate map keys in the Claims Set (see {{Section 5.6 of CBOR}}).
```

> > Note: Requiring an integer that is unique per decoy within the entire CWT ensures that there are no duplicate map keys in the Claims Set (see {{Section 5.6 of CBOR}}).

Reviewer: ```suggestion
> Note: Requiring an integer that is unique per decoy within the entire CWT ensures that there are no duplicate map keys in the Claims Set (see {{Section 5.6 of RFC8949}}).
```

> An Issuer MUST NOT issue an SD-CWT if the preissued Claims set contains duplicated keys.

Reviewer: ```suggestion
An Issuer MUST NOT issue an SD-CWT if the pre-issued Claims Set contains duplicated map keys.
```

> An Issuer MUST NOT issue an SD-CWT if the pre-issued Claims Set contains duplicated map keys.

Reviewer: ```suggestion
An Issuer MUST NOT issue an SD-CWT if the pre-issued Claims Set contains duplicate map keys.
```
